Ugizwenayo-Divine/Backend-Web-Development
# (chore): setup project

- setting up my project
#### What does this PR do?
- Setup the Backend-Web-Development project
#### Description of Task to be completed?
- create the project folder
- create package.json file by `npm init --yes`
- create index.js file which contains the server
#### How should this be manually tested?
- After cloning the project repo, run `npm install` inside the cloned repo folder
- Finally, run the following commands respectively:
`git fetch`
`git checkout  setup-project`
`npm install`
`npm run start`
- In  your browser visit `http://localhost:3000`
then you will see "Welcome to Backend Part!!!" message
#### Any background context you want to provide?
- You must have node  installed locally on your computer version 6 or above
- You must have any code editor
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
